components/converter/ConverterContent.js

```diff
@@ -1,57 +0,0 @@
-import React from 'react'
-import styled from 'styled-components'
-import { breakpoint } from 'lib/microsite-logic'
-import { useWeb3Connect } from 'lib/web3-connect'
-import Balance from './Balance'
-import Form from './Form'
-import Info from './Info'
-import Providers from './Providers'
```

Comment:
Unused file as far as I could see, these paths were invalid so removed the whole thing.